apps/bfd-model/bfd-model-rif/src/main/java/gov/cms/bfd/model/rif/parse/RifParsingUtils.java

```diff
@@ -269,4 +270,21 @@ public static Character parseCharacter(String charText) {
       return Optional.of(parseCharacter(charText));
     }
   }
+
+  /**
+   * @param string tring to parse
+   * @return an {@link String} with a null representation stripped from it if the input has a null
+   *     field representation.
+   */
+  public static String removeInvalidCharacters(String string) {
+    String newString = string;
+
+    // Null Field
+    String nullFields = "\\0";
+    if (!Strings.isNullOrEmpty(newString)) {
+      if (newString.contains(nullFields)) newString = newString.replace(nullFields, "");
+    }
+
+    return newString;
+  }
```

Comment:
I'd suggest a slight wording tweak to clarify your intent here:
```suggestion
  /**
   * @param string the {@link String} to parse
   * @return a copy of the specified {@link String} but with any/all null bytes it contained removed
   */
  public static String removeInvalidCharacters(String string) {
    String newString = string;

    String nullByte = "\\0";
    if (!Strings.isNullOrEmpty(newString)) {
      if (newString.contains(nullByte)) newString = newString.replace(nullByte, "");
    }

    return newString;
  }
```